docs/source/index.rst

```diff
@@ -27,10 +27,16 @@ This is the `Chainer <https://chainer.org>`_ documentation.
    comparison
    license
 
-
 Indices and tables
 ==================
 
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Other Support
```

Comment:
Also it's better to fix the stylesheet to make it easy to distinguish between internal and external links.

`<a>` elements in the navigation menu have `internal` and `external` classes respectively.